crates/qt-build-utils/src/utils.rs

```diff
@@ -14,3 +16,16 @@ pub(crate) fn is_apple_target() -> bool {
 pub(crate) fn is_emscripten_target() -> bool {
     std::env::var("CARGO_CFG_TARGET_OS") == Ok("emscripten".to_owned())
 }
+
+/// Wrap a command in a native subshell
+pub(crate) fn native_shell_command(command: &str) -> Command {
+    let mut result: Command;
+    if cfg!(target_os = "windows") {
+        result = Command::new("cmd");
+        result.args(["/C", command]);
+    } else {
```

Comment:
Please use an `else if` that actively lists the operating systems that we expect to have an `sh` binary and fall back to a plain `Command` in the other cases, as otherwise this will break on any OS that does not have an `sh` binary.